src/Explorer/Controls/Settings/SettingsUtils.test.tsx

```diff
@@ -139,3 +140,10 @@ describe("SettingsUtils", () => {
     expect(notification.type).toEqual(MongoNotificationType.Error);
   });
 });
+
+it("isIndexingTransforming", () => {
+  expect(isIndexTransforming(undefined)).toEqual(false);
+  expect(isIndexTransforming(0)).toEqual(true);
+  expect(isIndexTransforming(90)).toEqual(true);
+  expect(isIndexTransforming(100)).toEqual(false);
+});
```

Comment:
What about `isIndexTranforming(-1)`?

Reply:
index transform is a % value returned from the backend. So 0<= index transform progress% <=100